java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBStateStoreIT.java

```diff
@@ -343,9 +375,9 @@ public void shouldReturnOnlyActiveFilesWithNoJobId() throws StateStoreException
         public void shouldReturnOnlyActiveFilesWithNoJobIdWhenPaging() throws StateStoreException {
             // Given
             Schema schema = schemaWithSingleRowKeyType(new LongType());
-            StateStore dynamoDBStateStore = getStateStore(schema);
+            StateStore dynamoDBStateStore = getStateStoreWithPageLimit(schema, 10);
```

Comment:
It would be nice to add more tests for the paging behaviour. It'd be ideal if we could test this in DynamoDBUtilsPagingIT, although that would mean setting up a test table with a slightly more complex structure so that we can page through records differentiated by a range key (since a query requires an exact match on the hash key).

Reply:
Added tests for this in `DynamoDBUtilsPagingIT.RunningQuery`